CHANGELOG.md

```diff
@@ -157,11 +246,12 @@ See [MIGRATION.md](MIGRATION.md) for detailed upgrade instructions.
 
 ## Testing
 
-Version 1.0 has been tested on:
+Version 1.1.0 has been tested on:
 - **Hardware:** AMD Instinct MI350X, MI300X, MI250X, MI210
 - **Configuration:** 8-GPU systems with XGMI interconnect
 - **ROCm Version:** 6.3.0
-- **Test Coverage:** 26+ functions tested across 6 test suites
+- **Environments:** Bare metal and ROCm PyTorch containers
```

Comment:
Suggest adding something on Python environments, e.g.
- **Python setup:** Prefer conda for lib support, otherwise requires  python install amd_smi (see below)